amp/WEB-INF/src/org/digijava/module/aim/action/CompareActivityVersions.java

```diff
@@ -98,356 +100,358 @@ public ActionForward compare(ActionMapping mapping, ActionForm form, HttpServlet
-                                Object auxValue2 = auxVersionable2.getValue() != null ? auxVersionable2.getValue() : "";
-                                if (!auxValue2.equals(auxValue1)) {
-                                    // Check if the object was added in the
-                                    // previous iteration.
-                                    if (!auxList.contains(auxVersionable2)) {
-                                        CompareOutput auxOutput = new CompareOutput(auxAnnotation.fieldTitle(),
-                                                new String[] {
-                                                        ActivityVersionUtil.generateFormattedOutput(request,
-                                                                auxVersionable1.getOutput()),
-                                                        ActivityVersionUtil.generateFormattedOutput(request,
-                                                                auxVersionable2.getOutput()) }, fields[i],
-                                                new Object[] { auxObject1, auxObject2 }, false, false);
-                                        vForm.getOutputCollection().add(auxOutput);
-                                    }
-                                }
-                            }
-                        }
-                        if (coincidence == 0) {
-                            // Check if the object was added in the
-                            // previous iteration.
-                            if (!auxList.contains(auxVersionable2)) {
-                                CompareOutput auxOutput = new CompareOutput(auxAnnotation.fieldTitle(), new String[] {
-                                        "",
-                                        ActivityVersionUtil.generateFormattedOutput(request, auxVersionable2
-                                                .getOutput()) }, fields[i], new Object[] { null, auxObject2 }, false,
-                                        false);
-                                vForm.getOutputCollection().add(auxOutput);
-                            }
-                        }
-                    }
-                } else {
-                    output.setStringOutput(new String[] { auxResult1 != null ? auxResult1.toString() : "",
-                            auxResult2 != null ? auxResult2.toString() : "" });
-                    output.setDescriptionOutput(auxAnnotation.fieldTitle());
-                    vForm.getOutputCollection().add(output);
-                }
-            }
-        }
-
-        Map<String, List<CompareOutput>> outputGroupped = groupOutputCollection (vForm.getOutputCollection());
-        modifyFundingOutputs (outputGroupped);
-        vForm.setOutputCollectionGrouped(outputGroupped);
+//        vForm.setActivityOne((AmpActivityVersion) session.load(AmpActivityVersion.class, vForm.getActivityOneId()));
```

Comment:
same as before. code that is not in use anymore, should be removed. For traceability we have GitHub that we can easily see the history there